session/session.go

```diff
@@ -149,21 +149,22 @@ func New(cfg Config) (*session, error) {
 	return sess, nil
 }
 
-func initContacts(contactsType, contactsURL string, wb perun.WalletBackend, own perun.Peer) (perun.Contacts, error) {
-	if contactsType != "yaml" {
-		return nil, perun.ErrUnsupportedContactsType
+func initIDProvider(idProviderType, idProviderURL string, wb perun.WalletBackend, own perun.PeerID) (
+	perun.IDProvider, error) {
+	if idProviderType != "yaml" {
+		return nil, perun.ErrUnsupportedIDProviderType
 	}
-	contacts, err := contactsyaml.New(contactsURL, wb)
+	idProvider, err := local.NewIDprovider(idProviderURL, wb)
 	if err != nil {
 		return nil, err
 	}
 
 	own.Alias = perun.OwnAlias
-	err = contacts.Write(perun.OwnAlias, own)
+	err = idProvider.Write(perun.OwnAlias, own)
 	if err != nil && !errors.Is(err, perun.ErrPeerExists) {
-		return nil, errors.Wrap(err, "registering own user in contacts")
+		return nil, errors.Wrap(err, "registering own user in idProvider")
```

Comment:
```suggestion
		return nil, errors.Wrap(err, "registering own user in ID provider")
```

ID provider as two separate word when using in comments or error messages.